back-end/resources/room_config.json

```diff
@@ -175,48 +175,44 @@
                     {
                       "component_id": "mainSwitch",
                       "comparison": "eq",
-                      "value": true
+                      "value": false
                     },
                     {
-                      "component_id": "greenSwitch",
+                      "component_id": "slider1",
                       "comparison": "eq",
-                      "value": true
+                      "value": 100
                     },
                     {
-                      "component_id": "redSwitch",
+                      "component_id": "slider2",
                       "comparison": "eq",
-                      "value": true
+                      "value": 100
                     },
                     {
-                      "component_id": "slider2",
-                      "comparison": "lt",
-                      "value": 30
+                      "component_id": "slider3",
+                      "comparison": "eq",
+                      "value": 0
                     }
                   ]
                 }
+              }, {
+                "type": "rule",
```

Comment:
let's not make it dependent on the telephone

Reply:
I will make a new configuration